### PR TITLE
Review the Style Guide

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This is to review the contents of the Ballerina Style Guide.

Fixes https://github.com/ballerina-platform/ballerina-lang/issues/15974

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: docs/style_guide/README.md [hunk 1]
```diff
@@ -1,49 +1,48 @@
-# Ballerina style guide
+# Ballerina Style Guide
 
-Ballerina Style Guide aims at maintaining a standard coding style among Ballerina community. Therefore, Ballerina IDE plugins and tools follow this guide when providing code formatting and other features. 
+The Ballerina Style Guide aims at maintaining a standard coding style among the Ballerina community. Therefore, code formatting and the other features provided by the Ballerina IDE plugins and tools are based on this guide.
```
Comment (NipunaMarcus) [inline docs/style_guide/README.md]: can we change the above sentence to say something like bellow?
```suggestion
The Ballerina Style Guide aims at maintaining a standard coding style among the Ballerina community. Therefore, code formatting tools are based on this guide.
```

Comment (praneesha) [reply docs/style_guide/README.md]: Fixed.

File: docs/style_guide/README.md [hunk 2]
```diff
@@ -1,49 +1,48 @@
-# Ballerina style guide
+# Ballerina Style Guide
 
-Ballerina Style Guide aims at maintaining a standard coding style among Ballerina community. Therefore, Ballerina IDE plugins and tools follow this guide when providing code formatting and other features. 
+The Ballerina Style Guide aims at maintaining a standard coding style among the Ballerina community. Therefore, Therefore, the code formatting tools are based on this guide.
```
Comment (NipunaMarcus) [inline docs/style_guide/README.md]:
```suggestion
The Ballerina Style Guide aims at maintaining a standard coding style among the Ballerina community. Therefore, the code formatting tools are based on this guide.
```

File: docs/style_guide/README.md [hunk 3]
```diff
@@ -1,49 +1,48 @@
...
-    Don'ts
+    **Don'ts**
     ```ballerina
     public   function   getFullName()   returns   string {
         string   fullName = "john doe";
         return     fullName;
     }
     ```
-  Few exceptions for this rule are
+  Few exceptions for this rule are;
```
Comment (NipunaMarcus) [inline docs/style_guide/README.md]:
```suggestion
  Few exceptions for this rule are
```

File: docs/style_guide/definitions.md [hunk 4]
```diff
@@ -56,9 +56,9 @@
   ```ballerina
   function Person.getName() {}
   ```
-* If function needs to be split in to new lines, due to exceeding max line length
+* If the function needs to be split in to new lines due to it exceeding the max line length,
   - can break lines from the parameter list by moving a parameter value only to a 
-    new line and indent it with four spaces from the function start position.
+    new line and indenting it with four spaces from the starting position of the function.
     
     Examples,
```
Comment (NipunaMarcus) [inline docs/style_guide/definitions.md]:
```suggestion
    Example,
```

File: docs/style_guide/README.md (hunk 3 above)
Comment (praneesha) [reply docs/style_guide/README.md]: Fixed as "Few exceptions for this rule are:".

File: docs/style_guide/definitions.md (hunk 4 above)
Comment (praneesha) [reply docs/style_guide/definitions.md]: Fixed.

**Pull request merged**